pluribus/games/short_deck/state.py

```diff
@@ -80,6 +80,7 @@ def __init__(
             "show_down": player_i_order,
             "terminal": player_i_order,
         }
+        self._skip_counter = 0
```

Comment:
Old behavior allowed for skips to be appended before the player who has folded had been reached. It might have worked, but info sets had the strange behavior that actions were added after an action was made, and skips were added before the player was skipped.